src/CalculationClasses/MultiLayer/TransmissionMultiLayer_v2.m

```diff
@@ -0,0 +1,158 @@
+classdef TransmissionMultiLayer_v2<BaseMultiLayer
```

Comment:
What is the difference between v2 and original class? Why do we need the second?